src/components/application/index.js

```diff
@@ -1,23 +1,365 @@
+                      highlight={gradYearError.highlight}
+                      memo={gradYearError.message ? gradYearError.message : null}
+                      onChange={value => this.onFormInputChange('gradYear', value)}
+                      options={gradYears}
+                  />
+                  <FormInput
+                      className="app-age"
+                      value={ age }
+                      placeholder="Age"
+                      highlight={ageError.highlight}
+                      memo={ageError.message ? ageError.message : null}
+                      onChange={e => this.onFormInputChange('age', e.target.value)}
+                  />
+                  <FormInput
+                      className="app-ethnicity"
+                      value={ ethnicity }
+                      placeholder="Ethnicity"
+                      highlight={ethnicityError.highlight}
+                      memo={ethnicityError.message ? ethnicityError.message : null}
+                      onChange={e => this.onFormInputChange('ethnicity', e.target.value)}
+                  />
+                  <FormInput
+                      className="app-num-prev-hackathons"
+                      value={ numPrevHackathons }
+                      placeholder="Number of previous hackathons"
+                      highlight={numPrevHackathonsError.highlight}
+                      memo={numPrevHackathonsError.message ? numPrevHackathonsError.message : null}
+                      onChange={e => this.onFormInputChange('numPrevHackathons', e.target.value)}
+                  />
+                  <FormInput
+                      className="app-personal-website"
+                      value={ personalWebsite }
+                      placeholder="https://"
+                      highlight={personalWebsiteError.highlight}
+                      memo={personalWebsiteError.message ? personalWebsiteError.message : null}
+                      onChange={e => this.onFormInputChange('personalWebsite', e.target.value)}
+                  />
+                  <FormInput
+                      className="app-github-username"
+                      value={ githubUsername }
+                      placeholder="Github username"
+                      highlight={githubUsernameError.highlight}
+                      memo={githubUsernameError.message ? githubUsernameError.message : null}
+                      onChange={e => this.onFormInputChange('githubUsername', e.target.value)}
+                  />
+                  <input
+                      className="app-resume"
+                      name="resume"
+                      type="file"
+                      accept=".pdf,.docx,.doc"
+                      onChange={e => this.onFormInputChange('resume', e.target.files[0])} />
+                  <Button
+                      backgroundColor="bg-wh-pink"
+                      onClick={ this.onSubmitApp.bind(this) }
```

Comment:
remove `.bind` and turn `onSubmitApp` into a fat arrow function https://www.sitepoint.com/es6-arrow-functions-new-fat-concise-syntax-javascript/